wiki/People/Beatmap_Nomination_Group/Rules/en.md

```diff
@@ -0,0 +1,57 @@
+# Beatmap Nominator Rules
```

Comment:
give me an example on how to do that i'm having a hard time grasping the things you want to have changed here